src/vtbackend/StatusLineBuilder.cpp

```diff
@@ -420,9 +434,33 @@ struct VTSerializer
 
     std::string visit(StatusLineDefinitions::VTType const&) { return std::format("{}", vt.terminalId()); }
 
-    std::string visit(StatusLineDefinitions::Tabs const&)
+    std::string visit(StatusLineDefinitions::Tabs const& tabs)
     {
-        return std::format("{}", vt.guiTabInfoForStatusLine());
+        auto const tabsInfo = vt.guiTabsInfoForStatusLine();
+
+        std::string fragment;
+        for (size_t position: std::views::iota(1u, tabsInfo.tabCount + 1))
```

Comment:
we can use crispy::times here

Reply:
@Yaraslaut Yeah, I tried making more use of standard library. What do you think? `crispy::times` comes from a time where not even range-v3 was used :)